lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+  def initialize
+    @dishes = []
+  end
+
+  def dishes
+    @dishes =
+    [
+      { name: 'Burger', price: 10.95  },
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.